app/controllers/guests_controller.rb

```diff
@@ -0,0 +1,6 @@
+class GuestsController < ApplicationController
+  def show
+    @guests = Guest.find(params[:id]) 
```

Comment:
Just to not get confused, it might be helpful to name this `@guest` because it's just one single guest that's retrieve from the find method